src/ActionStatusType.php

```diff
@@ -17,36 +17,6 @@
  */
 class ActionStatusType extends BaseType implements ActionStatusTypeContract, EnumerationContract, IntangibleContract, StatusEnumerationContract, ThingContract
 {
-    /**
```

Comment:
Could you check these constants? As these aren't the only ones and always all constants I assume that it's a change in the JSON-LD we consume or an error in the generator. 🤔
In case they don't appear in the JSON-LD anymore - fine. Otherwise the generator should be adjusted.

Right now this would be a breaking change. In case the constants can be added back again this should be possible as minor.

Reply:
After digging into this my clever solution is just breaking more things. So I'm going to try a different angle here. Overall it should ultimately be cleaner with this alternative I'm thinking up.

Instead of mutating the source data to match what we expected (aka older formats) we'll modify output data.

Reply:
Fixed up the generator properly this time - the constants are happy again!